src/frontend/js/components/DashboardAddressesManagement/index.tsx

```diff
@@ -60,32 +60,33 @@ export const DashboardAddressesManagement = ({
     });
   }, [addresses.items]);
 
-  if (isLoading) {
-    return <Spinner />;
-  }
-
   return (
     <DashboardCard header={<FormattedMessage {...messages.header} />}>
       <div className="dashboard-addresses">
-        {error && <Banner message={error} type={BannerType.ERROR} rounded />}
-        {!error && addressesList.length === 0 && (
-          <p className="dashboard-addresses__empty">
-            <FormattedMessage {...messages.emptyList} />
-          </p>
+        {isLoading && <Spinner />}
+        {!isLoading && (
+          <>
+            {error && <Banner message={error} type={BannerType.ERROR} rounded />}
+            {!error && addressesList.length === 0 && (
+              <p className="dashboard-addresses__empty">
+                <FormattedMessage {...messages.emptyList} />
+              </p>
+            )}
+            {addressesList.map((address) => (
```

Comment:
Address list should be only displayed if there is no error right ?
```suggestion
            {!error && addressesList.map((address) => (
```